% Template for PLoS
% Version 3.5 March 2018
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended
% to minimize problems and delays during our production
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% Once your paper is accepted for publication,
% PLEASE REMOVE ALL TRACKED CHANGES in this file
% and leave only the final text of your manuscript.
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file.
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission.
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.
%
% Do not include text that is not math in the math environment.
%
% Please add line breaks to long display equations when possible in order to fit size of the column.
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
% \usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace}
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
% \bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother



% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}






\usepackage{forarray}
\usepackage{xstring}
\newcommand{\getIndex}[2]{
  \ForEach{,}{\IfEq{#1}{\thislevelitem}{\number\thislevelcount\ExitForEach}{}}{#2}
}

\setcounter{secnumdepth}{0}

\newcommand{\getAff}[1]{
  \getIndex{#1}{}
}

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Prediction of PIK3CA mutation with gene expression} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Jun Kang\textsuperscript{\getAff{Department of Hospital Pathology, Seoul St.~Mary's Hospital, College of
Medicine, The Catholic University of Korea, Seoul, South Korea}}\textsuperscript{*}\\
\bigskip
\bigskip
* Corresponding author: jkang.alien@gmail.com\\
\end{flushleft}
% Please keep the abstract below 300 words

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step.
% Author Summary not valid for PLOS ONE submissions.

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

\begin{itemize}
\item
  Brevity
\item
  Logic and clarity
\item
  Clean typing
\item
  The problem

  \begin{itemize}
  \tightlist
  \item
    PIK3CA mutation in selecting drug
  \end{itemize}
\end{itemize}

Breast cancer with PIK3CA mutation has been approved to use PIK3CA
inhibitor in hormone receptor positive HER2 negative subtype. {[}1{]}
Prediction of PIK3CA mutation was done by gene expression data of TCGA.

\begin{itemize}
\tightlist
\item
  Predicting mutation by gene expression data

  \begin{itemize}
  \tightlist
  \item
    What for?\\
  \item
    Feasible?\\
  \item
    Cost?
  \end{itemize}
\item
  Varing frquency of PIK3CA mutation across cancer types

  \begin{itemize}
  \tightlist
  \item
    High in endometrial breast\\
  \item
    not common other cancer type
  \end{itemize}
\end{itemize}

Universial prediction using gene expression data across cancer type for
certain kind of mutation. It's clinically useless now, but we want to
explore the possibilities of the PIK3CA mutation prediction.

\begin{itemize}
\item
  RNAseq can be widely used. The mutation status directly prediction
\item
  Previous study {[}2{]}
\item
  Prediction of aberant activation of a certain pathway vs a specific
  mutation
\item
  The proposed solution
\end{itemize}

Prediction RAS pathway activation with gene expression data was done in
previous study. {[}2{]} They trained pancancer The Cancer Genome Atlas
(TCGA) data with a supervised elastic net penalized logistic regression
classifier with stochastic gradient descent. The performance of their
model was 84\% with an area under the receiver operating characteristic
(AUROC) curve and 63\% with an area under the precision recall (AUPR)
curve. We applied their modeling methods in prediction PIK3CA mutation.

\hypertarget{materials-and-methods}{%
\section{Materials and Methods}\label{materials-and-methods}}

\hypertarget{dataset}{%
\subsection{Dataset}\label{dataset}}

We used TCGA pancancer dataset. PIK3CA mutation data was get using cgdsr
rpackage.{[}3{]} Gene expression data was get from GDAC firehose using
RTCGAToolbox R package. {[}4{]} ER immunostain postive HER2 immunostain
negative and/or SISH negative breast cancers are included. Data of
invasive ductal carcinomas were used for training set and data of
invasive lobular carcinoma were used for test set. Number of
observations were 530 in training set and 188 in test set.

\hypertarget{modeling}{%
\subsection{Modeling}\label{modeling}}

To narrow down potential predictors, Genes with a large the median
absolute deviation (more than third-quartiles) were selected. 5000 out
of 20502 genes were included in the modeling process. Yeo-Johnson
transformation was done to correct skewness. Centering and scaling were
done. All preprocessing was done using recipe r package. {[}5{]}
Penalized logistic regression was applied to prediction modeling.
10-fold cross-validation with targe variable stratification was done
over the hyperparameter grid: \(\lambda\) \{\(10^{-5}\),
\(10^{-4}\),\(10^{-3}\),\(10^{-2}\),\(10^{-1}\), \(10^{0}\)\},
\(\alpha\) \{0.0, 0.25, 0.5, 0.75\}. Model performance was evealuated
with the area under receiver operating characteristic (ROC) and area
under the ROC (AUROC) and precision recall (PR) and area under curve
(AUPR). The modelin process was done with tidymodels rpackage. {[}6{]}

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{selecting-model}{%
\subsection{Selecting model}\label{selecting-model}}

Cross-validation\\
The model showed best performance at lambda = 0.01 and alpha = 1.0
(Ridge regression).

\hypertarget{prediction-performance}{%
\subsection{Prediction performance}\label{prediction-performance}}

The trainset AUROC was 0.93 and the testset AUROC was 0.84. The AUPR of
trainset was 0.66 and the testset AUPR was 0.39. (Figure 1A)

\hypertarget{performance-of-each-cancer-type}{%
\subsection{Performance of each cancer
type}\label{performance-of-each-cancer-type}}

Because tht prevalence of PIK3CA mutation is vary across the cancer
type, the performance of each cancer type was investigated. The AUROC
and AUPR were positively correlated between train set and test set in
cancer type subanalysis. (Figure 1B) The AUPR was high in cancer type
with high PIK3CA mutation rate such as colon, brest, Uterus cancer
types. The AUROC did not correlated with PIK3CA mutation rate of each
cancer types. (Figure 1C)

\hypertarget{important-predictors}{%
\subsection{Important predictors}\label{important-predictors}}

Figure 2 shows top 30 important predictors. IGF1R mRNA expression was
the strongest negative predictor and PTEN was the strongest positive
predictor. Both genes and PIK3CA are key players in tyrosin kanase
pathway. The cancer type was important predictors. Some cancer types
including uterine carcinosarcoma (UCS), bladder urothelial carcinoma
(BLCA), pancreatic adenocarcinoma (PAAD), lymphoid neoplasm diffuse
large B-cell lymphoma (DLBC) are strongest predictors.

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

Our model showed good performance to predict PIK3CA mutation of various
cancer types.

\begin{itemize}
\item
  Main message answers the question and main supporting evidence
\item
  Critical assessment opinions on any shortcomings in study design
\end{itemize}

In this study, the split method was done for testset instead of
indepedent testset. Because there is no standard analysis method for the
RNA-seq gene expression quantification, it is difficult to find gene
expression data same as TGCA data. Developing standard analysis method
for gene expression data is neccessory to apply this prediction model.

In this study, the prevalence of PIK3CA mutation was 11\%. The low
prevalence of PIK3CA results in imbalaced dataset which makes the
prediction difficult. The AUROC is optimistic in terms of performance.
The AUPR is more informative than AUROC on imbalaced datasets. {[}7{]}

\begin{itemize}
\item
  limitations in methods

  \begin{itemize}
  \tightlist
  \item
    Case imbalance\\
  \item
    flaws in analysis\\
  \item
    validity of assumption
  \end{itemize}
\item
  Comparison with other studies where inconsistencies are discussed\\
\item
  Evaluate the results - not the authors
\end{itemize}

PIK3CA prediction model performed better than RAS activation prediction
model in terms of AUROC. AUPR and the PIK3CA mutation rate
interpretation. {[}7{]}

\begin{itemize}
\tightlist
\item
  Conclusions comments on possible biological or clinical implications
  and suggestions for further research.
\end{itemize}

\hypertarget{figure-legends}{%
\section{Figure legends}\label{figure-legends}}

\begin{itemize}
\tightlist
\item
  Figure 1. Summary of modeling results
\end{itemize}

\begin{enumerate}
\def\labelenumi{(\Alph{enumi})}
\tightlist
\item
  Left: receiver operating characteristic (ROC) curve right: precision
  recall (PR) curve of trainset and testset. The horizontal green line
  is the PIK3CA mutation rate (0.11) (B) Correlation between trainset
  and testset of area under receiver operating characteristic curve
  (AUROC) and area under precision recall curve (AUPR) among cancer
  type. The abbreviations are explained in S1 appendix. (C) Correlation
  between the PIK3CA mutation rate of area under receiver operating
  characteristic curve (AUROC) and area under precision recall curve
  (AUPR).
\end{enumerate}

\begin{itemize}
\tightlist
\item
  Figure 2. Coefficients of model
\end{itemize}

\begin{enumerate}
\def\labelenumi{(\Alph{enumi})}
\tightlist
\item
  Top 30 high coefficients of mRNA. (B) Coefficients of cancer types.
  The abbreviations are explained in S1 appendix.
\end{enumerate}

\hypertarget{supporting-information}{%
\section{Supporting information}\label{supporting-information}}

\begin{itemize}
\tightlist
\item
  S1 Appendix.\\
\item
  S2 Figure.\\
\item
  S1 Table.\\
\item
  S2 Table.\\
\item
  S3 Table.
\end{itemize}

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-andre_2019_alpelisib_nengljmeda}{}%
1. André F, Ciruelos E, Rubovszky G, Campone M, Loibl S, Rugo HS, et al.
Alpelisib for PIK3CA-Mutated, Hormone ReceptorPositive Advanced Breast
Cancer. New England Journal of Medicine. Massachusetts Medical Society;
2019;380: 1929--1940.
doi:\href{https://doi.org/10.1056/NEJMoa1813904}{10.1056/NEJMoa1813904}

\leavevmode\hypertarget{ref-way_2018_machine_cellreports}{}%
2. Way GP, Sanchez-Vega F, La K, Armenia J, Chatila WK, Luna A, et al.
Machine Learning Detects Pan-cancer Ras Pathway Activation in The Cancer
Genome Atlas. Cell Reports. Elsevier; 2018;23: 172--180.e3.
doi:\href{https://doi.org/10.1016/j.celrep.2018.03.046}{10.1016/j.celrep.2018.03.046}

\leavevmode\hypertarget{ref-jacobsenCgdsrRbasedAPI2019}{}%
3. Jacobsen A, Luna A. Cgdsr: R-based API for accessing the MSKCC cancer
genomics data server (CGDS). 2019.

\leavevmode\hypertarget{ref-samurRTCGAToolboxNewTool2014}{}%
4. Samur MK. RTCGAToolbox: A new tool for exporting TCGA Firehose data.
PLoS One. 2014;9(9):e106397.

\leavevmode\hypertarget{ref-kuhnRecipesPreprocessingTools2020}{}%
5. Kuhn M, Wickham H. Recipes: Preprocessing tools to create design
matrices. 2020.

\leavevmode\hypertarget{ref-kuhnTidymodelsEasilyInstall2020}{}%
6. Kuhn M, Wickham H. Tidymodels: Easily install and load the
'tidymodels' packages. 2020.

\leavevmode\hypertarget{ref-saitoPrecisionRecallPlotMore2015}{}%
7. Saito T, Rehmsmeier M. The Precision-Recall Plot Is More Informative
than the ROC Plot When Evaluating Binary Classifiers on Imbalanced
Datasets. PLOS ONE. Public Library of Science; 10: e0118432.
doi:\href{https://doi.org/10.1371/journal.pone.0118432}{10.1371/journal.pone.0118432}

\nolinenumbers


\end{document}

